### ISSUE TITLE
Wrong cardinality on curves in CompositeCurve3D

### ISSUE THREAD

**Issue by ocastrup** [OCXStandard/OCX_Schema#103]
CompositeCurve3D can only contain 1 type of curve:
```XML
	<xs:complexType name="CompositeCurve3D_T">
		<xs:annotation>
			<xs:documentation>Type definition of a composite curve composed of a collection of Line3D, CircumArc3D and/or NURBS segments..</xs:documentation>
		</xs:annotation>
		<xs:complexContent>
			<xs:extension base="ocx:Curve3D_T">
				<xs:choice maxOccurs="unbounded">
					<xs:element ref="ocx:Line3D" minOccurs="0"/>
					<xs:element ref="ocx:CircumArc3D" minOccurs="0"/>
					<xs:element ref="ocx:NURBS3D" minOccurs="0" maxOccurs="unbounded"/>
					<xs:element ref="ocx:PolyLine3D" minOccurs="0">
						<xs:annotation>
							<xs:documentation>A list of 3D points defining a list of linear segments. Repeat the first position at the end to form a closed contour.</xs:documentation>
						</xs:annotation>
					</xs:element>
				</xs:choice>
			</xs:extension>
		</xs:complexContent>
```

**Comment by ocastrup** [comment]
Made changes to the CompositeCurve3D:
```XML
	<xs:complexType name="CompositeCurve3D_T">
		<xs:annotation>
			<xs:documentation>Type definition of a composite curve composed of a collection of Line3D, CircumArc3D and/or NURBS segments..</xs:documentation>
		</xs:annotation>
		<xs:complexContent>
			<xs:extension base="ocx:OpenCurve_T">
				<xs:choice>
					<xs:element ref="ocx:Line3D" maxOccurs="unbounded"/>
					<xs:element ref="ocx:CircumArc3D" maxOccurs="unbounded"/>
					<xs:element ref="ocx:NURBS3D" maxOccurs="unbounded"/>
					<xs:element ref="ocx:PolyLine3D" maxOccurs="unbounded">
						<xs:annotation>
							<xs:documentation>A list of 3D points defining a list of linear segments. Repeat the first position at the end to form a closed contour.</xs:documentation>
						</xs:annotation>
					</xs:element>
				</xs:choice>
			</xs:extension>
		</xs:complexContent>
```

**Comment by ocastrup** [comment]
Included in #104
